lib/hooks/useTime.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { formatter } from '../utils';
+
+const useTime = () => {
+  const SunCalc = require('suncalc');
+  const [currentTime, setCurrentTime] = useState(formatter.format(Date.now()));
+  const [currentTimeNoFilter, setCurrentTimeNoFilter] = useState(Date.now());
```

Comment:
Perhaps this `useTime` hook should deal solely with "no filter" time, or unformatted time. This way, it can be most easily used by the largest possible group of consumers.